lib/ioki/model/operator/public_transport_stop.rb

```diff
@@ -0,0 +1,85 @@
+# frozen_string_literal: true
+
+module Ioki
+  module Model
+    module Operator
+      class PublicTransportStop < Base
+        attribute :type,
+                  on:   :read,
+                  type: :string
+
+        attribute :city,
+                  type: :string,
+                  on:   :read
+
+        attribute :country,
+                  type: :string,
+                  on:   :read
+
+        attribute :county,
+                  type: :string,
+                  on:   :read
+
+        attribute :current_arrival,
+                  type: :string,
+                  on:   :read
+
+        attribute :current_departure,
+                  type: :string,
+                  on:   :read
+
+        attribute :current_platform,
+                  type: :string,
+                  on:   :read
+
+        attribute :dhid,
+                  type: :string,
+                  on:   :read
+
+        attribute :display_times,
```

Comment:
These could also be `DateTime` objects if you want them to be.

Reply:
type array is fine 👍